GliaWidgets/Sources/ViewController/Chat/ChatViewController.swift

```diff
@@ -262,12 +263,12 @@ extension ChatViewController {
             if chatViewModel.shouldSkipEnqueueingState {
                 return chatViewModel.activeEngagement != nil
                 ? .chat
-                : .secureTranscript
+                : .secureTranscript(allowsInput: true)
             } else {
                 return .chat
             }
-        case .transcript:
-            return .secureTranscript
+        case .transcript(let transcriptModel):
+            return .secureTranscript(allowsInput: transcriptModel.isSecureConversationsAvailable)
```

Comment:
Wouldnt it be slighty more "Swifty" if this is called `needsInput` rather?

Reply:
needsTextInput d1fc0441d3c41af49d93b72b4227be56ba84fc03.